### PR TITLE
Invitations: fix tablet view [DEPENDENCIES]

### PR DESCRIPTION
**Pull request by Pineirin**
Fix mobile view.
Add status icon.
Add extra space for user full name.
closes: #1595
depends on: https://github.com/inveniosoftware/invenio-communities/pull/681
depends on: https://github.com/inveniosoftware/invenio-requests/pull/238

Screenshot from 2022-05-23 14-59-42
Screenshot from 2022-05-23 14-59-50
Screenshot from 2022-05-23 14-59-58

### CODE REVIEW

File: invenio_app_rdm/theme/assets/semantic-ui/less/invenio_app_rdm/theme/elements/button.overrides [hunk 1]
```diff
@@ -56,7 +56,7 @@
 }
 
 .ui.button.fluid-mobile {
-  @media all and (max-width: @largestMobileScreen) {
+  @media all and (max-width: @largestTabletScreen) {
```
Comment (jennur) [inline invenio_app_rdm/theme/assets/semantic-ui/less/invenio_app_rdm/theme/elements/button.overrides:59]: Can we rather add another class for tablet `.fluid-tablet` with `@media screen and (min-width: @tabletBreakpoint) and (max-width: @largestTabletScreen)` , so that it's not affecting other cases where the button should not be fluid on tablet? Then apply both `.fluid-mobile` and `.fluid-tablet` to the button that should be fluid on both screen sizes?

File: invenio_app_rdm/theme/assets/semantic-ui/less/invenio_app_rdm/theme/collections/table.overrides [hunk 2]
```diff
@@ -22,6 +22,11 @@
       padding-top: 0;
     }
 
+    &:not(.unstackable) thead tr:not(.stackable-content) {
```
Comment (kpsherva) [inline invenio_app_rdm/theme/assets/semantic-ui/less/invenio_app_rdm/theme/collections/table.overrides:25]: what is the usage of the `.stackable-content` class ?

File: invenio_app_rdm/theme/assets/semantic-ui/less/invenio_app_rdm/theme/modules/dropdown.overrides [hunk 3]
```diff
@@ -66,14 +66,28 @@
...
-    min-width: 0em;
-    display: block;
-
-    & > .dropdown.icon {
-      float: right;
+.ui.dropdown {
+  &.fluid-tablet{
+    @media screen and (min-width: @tabletBreakpoint) and (max-width: @largestTabletScreen) {
+      width: 100%;
+      min-width: 0em;
```
Comment (kpsherva) [inline invenio_app_rdm/theme/assets/semantic-ui/less/invenio_app_rdm/theme/modules/dropdown.overrides:73]: I am not convinced for the min-width:0, I think we should have there a better value adjusted per device, otherwise all the dropdowns will have random minimal lengths based on their content

File: invenio_app_rdm/theme/assets/semantic-ui/less/invenio_app_rdm/theme/collections/table.overrides (hunk 2 above)
Comment (Pineirin) [reply invenio_app_rdm/theme/assets/semantic-ui/less/invenio_app_rdm/theme/collections/table.overrides:25]: It was to fix the bulk actions in the tablet, but I think that a merge avoided the reason it was included in the first place.

File: invenio_app_rdm/requests_ui/views/requests.py [hunk 4]
```diff
@@ -126,12 +126,12 @@ def user_dashboard_request_view(request, **kwargs):
 @pass_community
 def community_dashboard_request_view(request, community, **kwargs):
     """Community dashboard requests details view."""
-    request_type = request["type"]
-
     avatar = current_user_resources.users_service.links_item_tpl.expand(
         current_user
     )["avatar"]
 
+    request_type = request["type"]
```
Comment (kpsherva) [inline invenio_app_rdm/requests_ui/views/requests.py:133]: why is this moved below?

File: invenio_app_rdm/theme/assets/semantic-ui/less/invenio_app_rdm/theme/elements/button.overrides [hunk 5]
```diff
@@ -55,10 +55,19 @@
...
+    @media screen and (min-width: @tabletBreakpoint) and (max-width: @largestTabletScreen) {
+      width: 100%;
+      min-width: 0em;
+      display: block;
+    }
+  }
+  &.fluid-mobile {
+    @media all and (max-width: @largestMobileScreen) {
+      width: 100%;
+      min-width: 0em;
```
Comment (kpsherva) [inline invenio_app_rdm/theme/assets/semantic-ui/less/invenio_app_rdm/theme/elements/button.overrides:69]: I think I commented on that, but my comment was removed? 
I would apply min-width at least of 4em, 0 will cause the dropdowns to be very varied between them in length

File: invenio_app_rdm/theme/assets/semantic-ui/less/invenio_app_rdm/theme/elements/button.overrides [hunk 6]
```diff
@@ -55,10 +55,19 @@
...
-.ui.button.fluid-mobile {
-  @media all and (max-width: @largestMobileScreen) {
-    width: 100%;
-    min-width: 0em;
-    display: block;
+.ui.button{
+  &.fluid-tablet {
+    @media screen and (min-width: @tabletBreakpoint) and (max-width: @largestTabletScreen) {
+      width: 100%;
+      min-width: 0em;
```
Comment (kpsherva) [inline invenio_app_rdm/theme/assets/semantic-ui/less/invenio_app_rdm/theme/elements/button.overrides:62]: I think I commented on that, but my comment was removed? 
I would apply min-width at least of 4em, 0 will cause the dropdowns to be very varied between them in length

File: invenio_app_rdm/theme/assets/semantic-ui/less/invenio_app_rdm/theme/elements/button.overrides [hunk 7]
```diff
@@ -55,10 +55,19 @@
...
+      display: block;
+    }
+  }
+  &.fluid-mobile {
+    @media all and (max-width: @largestMobileScreen) {
+      width: 100%;
+      min-width: 0em;
+      display: block;
```
Comment (kpsherva) [inline invenio_app_rdm/theme/assets/semantic-ui/less/invenio_app_rdm/theme/elements/button.overrides:70]: it looks like fluid-tablet class is not needed ? since it is code duplication

I would replace it with 
```
&.fluid-responsive {
  @media screen and max-width: @largestTabletScreen {
 ```
 and there is no need to duplicate custom classes

File: invenio_app_rdm/theme/assets/semantic-ui/less/invenio_app_rdm/theme/modules/dropdown.overrides [hunk 8]
```diff
@@ -66,14 +66,26 @@
...
-  @media all and (max-width: @largestTabletScreen) {
-    width: 100%;
-    min-width: 0em;
-    display: block;
-
-    & > .dropdown.icon {
-      float: right;
+.ui.dropdown {
+  &.fluid-tablet{
+    @media screen and (min-width: @tabletBreakpoint) and (max-width: @largestTabletScreen) {
```
Comment (kpsherva) [inline invenio_app_rdm/theme/assets/semantic-ui/less/invenio_app_rdm/theme/modules/dropdown.overrides:71]: same duplication comment as above

File: invenio_app_rdm/requests_ui/views/requests.py (hunk 4 above)
Comment (Pineirin) [reply invenio_app_rdm/requests_ui/views/requests.py:133]: Merge issues.

**Review by kpsherva — APPROVED** [review]
